doc/fitting.rst

```diff
@@ -1,11 +0,0 @@
-``gempython.gemplotting.fitting`` --- Fitting tools
-===================================================
```

Comment:
won't removing these lines remove them from the documentation...?

Reply:
They are included in another way.